system-setup.py

```diff
@@ -41,7 +41,11 @@ def fedora(self):
 
     def macos(self):
         self.install_gnu_utils()
-        self.run("{PYTHON} {READIES}/bin/getredis -v 6 --force".format(PYTHON=self.python, READIES=READIES))
+        self.install("pkg-config")
```

Comment:
do we really need `pkg-config`?

Reply:
Adding pkg-config silences warnings on the command line. We don't need it
to compile, but do to silence warnings.